apps/website/src/components/show-and-tell/ShowAndTellEntry.tsx

```diff
@@ -191,6 +195,14 @@ const Content = ({ entry, isPresentationView }: ShowAndTellEntryProps) => {
     }
   }, [entry.text, entry.id]);
 
+  const { data: modComments, error } = trpc.showAndTell.getModComments.useQuery(
+    entry.id,
+  );
+
+  if (error) {
+    console.error("Failed to fetch mod comments:", error);
+  }
```

Comment:
I assume you made this a separate query 'cause fetching them with the posts was messy with types or something? If you can make it work in the main post data a bunch of the following comments will be obsolete.